modules/ROOT/pages/install-self-managed.adoc

```diff
@@ -11,37 +11,68 @@ Before installing Anypoint Runtime Fabric in a self-managed Kubernetes environme
 - Running Kubernetes version 1.14.x to 1.18.x.
 - Running an ingress controller to send external requests to applications.
 
-== Create a Runtime Fabric using Runtime Manager
-
-To install Runtime Fabric on a self-managed Kubernetes environment, first create a Runtime Fabric using Runtime Manager. This is required to obtain the activation data which is needed during installation.
-
-. From Anypoint Platform, select Runtime Manager.
-. Click *Runtime Fabrics*.
-. Click *Create Runtime Fabric*.
-. Enter the name of the new Runtime Fabric, then select one of the following options:
-+
-* Amazon Elastic Kubernetes Service
-* Azure Kubernetes Service
-
-. Click *Next*.
-. Review the *Support responsibility* disclaimer, then if you agree click *Continue*.
-+
-Runtime Manager creates the Runtime Fabric and displays the Activation State page. This page displays the activation data used to install Runtime Fabric on a Kubernetes service. Copy this data to the clipboard for use in the next section. 
-
-
-== Install Runtime Fabric on a Self-Managed Kubernetes Environment
-
-After creating a Runtime Fabric and getting the activation data, install Runtime Fabric into your Kubernetes service using the `rtfctl` command.
-
-[NOTE]
-====
-If your Kubernetes configuration is not located in the `~/.kube/config` directory, set the `KUBECONFIG` environment variable before running `rtfctl`:
-
-----
-export KUBECONFIG=<path-to-kubeconfig>
-----
-====
-
+== Network Configuration
+
+Before installint or using Runtime Fabric, ensure that the following ports and hostnames are configured correctly.
+
+=== Port Configuration
+
+To install or run Runtime Fabric, ensure that you have configured the following ports on you Kubernetes installation:
+
+[%header,cols="6*a"]
+.Port Configuration for Self-managed Kubernetes
+|===
+| Port | Layer 4 Protocol | Layer 5 Protocol | Source | Destination | Description
+| 443 | TCP | HTTPS | Internal | All nodes | Allow inbound requests to Mule runtime servers
```

Comment:
Should the source be "external" or "internet"?

Reply:
Fixed. Verified everything else matches table in the Beta Google Doc.